app/components/ProfileModal/index.js

```diff
@@ -87,16 +93,16 @@ const ProfileModal = ({
       onPressFunction: onPressManageSpecies,
       text: 'manage_species',
     },
-    {
-      media: 'cloud-upload-alt',
-      mediaType: 'icon',
-      text: 'back_up',
-    },
-    {
-      media: 'map-marked',
-      mediaType: 'icon',
-      text: 'manage_offline',
-    },
+    // {
+    //   media: 'cloud-upload-alt',
+    //   mediaType: 'icon',
+    //   text: 'all_backed_up',
+    // },
+    // {
+    //   media: 'map-marked',
+    //   mediaType: 'icon',
+    //   text: 'manage_offline',
+    // },
```

Comment:
I commented these menu items unless we really want to support them - both is accessible otherwise.

Reply:
Backup menu item is removed in activity log PR and manage offline maps can be commented as of now.